docs/API.md

```diff
@@ -115,91 +180,76 @@ const stamp = stampit().methods({
 stamp().amplify('BADF00D'); // value BADF00D is incorrect
 ```
 
-### stamp.refs()
+### stamp.props() and stamp.properties()
 
 Take n objects and add them to the references list of a new stamp. Creates new stamp.
-* @return {Object} stamp  The new stamp based on the original `this` stamp.
-
-It has an alias - `stamp.state()`. Deprecated.
+* `@return {Object} stamp` The new stamp based on the original `this` stamp.
 
 ```js
-const stamp = stampit().refs({
+const stamp = stampit().props({
   factor: 1
 });
 
 console.log(stamp().factor); // 1
 console.log(stamp({factor: 5}).factor); // 5
 ```
 
+
 ### stamp.init([arg1] [,arg2] [,arg3...])
 
 Take n functions, an array of functions, or n objects and add
 the functions to the initializers list of a new stamp. Creates new stamp.
-* @return {Object} stamp  The new stamp based on the original `this` stamp.
-
-If any of the init() functions return a Promise then the stamp will always be creating Promises 
-which resolve to the expected object instance.
-
-It has an alias - `stamp.enclose()`. Deprecated.
+* `@return {Object} stamp` The new stamp based on the original `this` stamp.
 
 Functions passed into `.init()` are called any time an
 object is instantiated. That happens when the stamp function
 is invoked, or when the `.create()` method is called.
 
-Each function receives the following object as the first argument:
+If an initializer returns a non-undefined value then it becomes the factory 
+instantiated value.
+
+Each function receives the following object as the second argument:
```

Comment:
No mention of what is the first argument. It's probably understandable from example below, but might be good to say: "Each function receives first argument of called factory function and second object argument like this:"